team/bundles/org.eclipse.jsch.ui/src/org/eclipse/jsch/internal/ui/messages.properties

```
@@ -89,6 +89,7 @@ CVSSSH2PreferencePage_144=Key Exchange &Methods
 CVSSSH2PreferencePage_145=MA&C Methods
 CVSSSH2PreferencePage_146=&SSH Agent
 CVSSSH2PreferencePage_147=Select preferred SSH Agent
+CVSSSH2PreferencePage_148=Key Size to Generate:
```

Let's keep it simple here:
```suggestion
CVSSSH2PreferencePage_148=Key size:
```